openmp-17.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: openmp-17
+  version: 17.0.5
+  epoch: 0
+  description: "LLVM OpenMP library"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
```

Comment:
nit: Wolfi-base already includes the `ca-certificates-bundle` in case you want to simplify it.

Reply:
wolf-base doesn't include  `ca-certificates-bundle` , its build-base which provides  `ca-certificates-bundle` 
 build-base -> wolfi-baselyout -> ca-certificates-bundle

Reply:
question: do we care about adding a package if another package already includes it ?

Reply:
I think we should remove them , its spread across various packages , there might be some reasoning behind it , i am not completely sure